sound/soc/intel/common/soc-acpi-intel-bxt-match.c

```diff
@@ -6,9 +6,43 @@
  *
  */
 
+#include <linux/dmi.h>
 #include <sound/soc-acpi.h>
 #include <sound/soc-acpi-intel-match.h>
 
+static unsigned long apl_machine_id;
+
+#define APL_TDF8532  1
+
+static int apl_tdf8532_quirk_cb(const struct dmi_system_id *id)
+{
+	apl_machine_id = APL_TDF8532;
```

Comment:
apl_machine_id = APL_RVP;
?

Reply:
yes, makes sense to change. More logical, thanks for the suggestion.